cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -113,8 +113,8 @@ set(config_module_list
 	#
 	# Logging
 	#
-	#modules/logger
-	modules/sdlog2
+	modules/logger
```

Comment:
yes, I'd like to limit this feature to boards with > 1MB of flash, but not sure if that's possible